dashboard/dashapi/jwt.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2017 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package dashapi
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"sync"
+	"time"
+)
+
+type expiringToken struct {
+	token      string
+	expiration time.Time
+}
+
+// Returns the unverified expiration value from the given JWT token.
+func extractJwtExpiration(token string) (time.Time, error) {
+	// https://datatracker.ietf.org/doc/html/rfc7519#section-3
+	pieces := strings.Split(token, ".")
+	if len(pieces) != 3 {
+		return time.Time{}, fmt.Errorf("unexpected number of JWT components %v", len(pieces))
+	}
+	decoded, err := base64.URLEncoding.WithPadding(base64.NoPadding).DecodeString(pieces[1])
```

Comment:
Docs say there is `RawURLEncoding`:
https://golang.org/pkg/encoding/base64/#pkg-variables
Or it's not present in the appengine version?